<launch>

  <node name="finger_vision1" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video8" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="finger_vision1" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="finger_vision2" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video6" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="finger_vision2" />
    <param name="io_method" value="mmap"/>
  </node>
  
</launch>
